ui-tests/.gitignore

```diff
@@ -1,7 +1,6 @@
 node_modules/
 /test-results/
 /playwright/.cache/
-/playwright-report
```

Comment:
wont't this mean that people will commit the reports when they run tests locally?

there was a massive index file checked in as part of this pr

Reply:
We need to keep this not gitignored so when the Playwright test is run in a GH Action, we can see the result. 

However, you are right that the report I ran shouldn't be added - I removed the HTML file from the PR.